pyproject.toml

```diff
@@ -73,3 +73,4 @@ load_macros_from_path = "dbt/macros"
 # that reference those variables if they are used in code that sqlfluff
 # lints
 get_s3_dependency_dir = "{% macro get_s3_dependency_dir() %}s3://bucket{% endmacro %}"
+insert_hyphens = "{% macro insert_hyphens(str) %}{% set _ = varargs %}'{{ str }}'{% endmacro %}"
```

Comment:
`{% set _ = varargs %}` is kind of goofy here, but we need to reference `varargs` or else dbt's jinja templater will complain about `insert_hyphens` being called with arbitrary positional arguments.